docs/faqs/rileylink-faqs.md

```diff
@@ -1,67 +1,67 @@
 
 
 ## RileyLink Questions
 
 ### What are the differences between the RileyLink Medtronic and Omnipod Antennas?
 
-There are two types of antennas for RileyLinks; each antenna is optimized for the pump you are using. Otherwise they are identical. See [Build Step 5: Radio-Link Options](../build/step5.md#radio-link-options) for other radio-links.  The OrangeLink has both antennas included in its design and can talk to either Medtronic or Omnipod.  The EmaLink requires selection for type of pump.
+There are two types of antennas for RileyLinks; each antenna is optimized for the pump you are using. Otherwise they are identical. See [Build Step 5: RileyLink Compatible Devices](../build/step5.md#rileylink-compatible-devices) for other devices.  The OrangeLink has both antennas included in its design and can talk to either Medtronic or Omnipod.  The EmaLink requires selection for type of pump.
 
 !!! note ""
     The color of the RileyLink circuit board in the photos below is irrelevant.
 
 ![components for RileyLink: case, board, battery; Medtronic on left, Omnipod on right](img/two-rl.png){width="650"}
 {align="center"}
 
-### What will happen if your radio-link has the wrong antenna?
+### What will happen if your RileyLink has the wrong antenna?
 
-You can *technically* use that radio-link with either pump on Loop. But, you will have significant frustrations and probably a lot of red loops. With mismatched antenna/pump, the radio-link needs to be very close (think inches) and in clear line-of-sight to pump/pod. This makes everyday living (and sleeping) a bit hard. If you use the appropriate-antenna-for-your-pump radio-link, the distances the pump/pod and radio-link can tolerate from each other is much more "real world" friendly and stable. The OrangeLink contains both antennas so will work with either pump. This may be a good choice if you like to switch between Medtronic and Omnipod.
+You can *technically* use that RileyLink with either pump on Loop. But, you will have significant frustrations and probably a lot of red loops. With mismatched antenna/pump, the device needs to be very close (think inches) and in clear line-of-sight to pump/pod. This makes everyday living (and sleeping) a bit hard. If you use the appropriate-antenna-for-your-pump device, the distances the pump/pod and RileyLink can tolerate from each other is much more "real world" friendly and stable. The OrangeLink contains both antennas so will work with either pump. This may be a good choice if you like to switch between Medtronic and Omnipod.
```

Comment:
This question and answer is not RileyLink specific, it's for any RileyLink compatible device with a single (wrong) antenna (i.e., anything but an OrangeLink). 

Saying "probably a lot of red loops" is overstated IMO. Perhaps a better & more realistic statement here would be "more yellow & red loops if you are not diligent about the placement of your device"